frontend/src/views/Landing/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Header from '../../components/navigation/Header';
+
+const Content = styled.div`
+  width: 80%;
+  margin: 10%;
+  text-align: center;
+  font-size: large;
+`;
+
+function LandingPage() {
```

Comment:
Is this used anywhere? If not, how do we incorporate it?

Reply:
In the future, I'd imagine it being incorporated using React Router (see [this example](https://github.com/withspectrum/spectrum/blob/alpha/admin/src/routes.js#L29)) which basically will allow us to have routes such as /landing, /login, /game, /results, etc. Right now we don't have that though, so I just included it at the bottom of the App component (you should see it if you scroll down on localhost:8080).

Reply:
Thanks for including that -- looks good.